server/controllers/scrums/put.js

```diff
@@ -1 +1,26 @@
-exports.put = (req, res) => res.send('single scrum put');
+const joi = require('joi');
+
+const updateScrum = require('./../../database/queries/updateScrum');
+
+exports.put = (request, response, next) => {
+  const schema = joi.object().keys({
+    name: joi.string(),
+  });
+  const { name } = request.body;
+  const result = joi.validate({ name }, schema);
+  const { scrumId } = request.params;
+
+  if (!result.error) {
+    updateScrum(name, scrumId)
+      .then((res) => {
+        console.log(res);
```

Comment:
Please remove all `console.log` from the code that you want to merge to the `master`

![bitmoji](https://render.bitstrips.com/v2/cpanel/fe4a1c10-cce9-4b4a-a43c-6e707afcc5d1-b481cd19-64ac-4cc9-8a0a-af105fe4206f-v1.png?transparent=1&palette=1&width=246)